src/toil/jobStores/googleJobStore.py

```diff
@@ -158,17 +158,26 @@ def destroy(self):
         except exceptions.NotFound:
             # just return if not connect to physical storage. Needed for idempotency
             return
+        
         try:
             self.bucket.delete(force=True)
-            # throws ValueError if bucket has more than 256 objects. Then we must delete manually
         except ValueError:
-            self.bucket.delete_blobs(self.bucket.list_blobs())
+            with self.storageClient.batch():
+                for blob in self.bucket.list_blobs():
+                    blob.delete()
             self.bucket.delete()
-            # if ^ throws a google.cloud.exceptions.Conflict, then we should have a deletion retry mechanism.
+        self.bucket = None
+        # try:
+        #     self.bucket.delete(force=True)
+        #     # throws ValueError if bucket has more than 256 objects. Then we must delete manually
+        # except ValueError:
+        #     self.bucket.delete_blobs(self.bucket.list_blobs())
+        #     self.bucket.delete()
+        #     # if ^ throws a google.cloud.exceptions.Conflict, then we should have a deletion retry mechanism.
```

Comment:
This old code should just be removed; we can always go back and get the old version from the Git history if we want it.